paasta_tools/mesos_maintenance.py

```diff
@@ -319,12 +332,17 @@ def datetime_to_nanoseconds(dt):
     return seconds_to_nanoseconds(int(dt.strftime("%s")))
 
 
-def build_start_maintenance_payload(hostnames):
+def build_start_maintenance_payload(hostnames, maint_type):
```

Comment:
Can we rename this function? It is a bit confusing calling `build_start_maintenance_payload` with a `main_type` of `stop_maintenance`.